server.go

```diff
@@ -208,6 +209,19 @@ func StatsHandler(h stats.Handler) ServerOption {
 	}
 }
 
+// UnknownStreamHandler returns a ServerOption that allows for adding a custom handler for unknown gRPC services/methods.
```

Comment:
About the naming, seems to me this is more like a `UnknownServiceHandler`. Or probably `DefaultHandler`?